File: snippets/visualbasic/VS_Snippets_CLR/ObjDispEx/VB/objdispexc.vb
﻿'<Snippet1>
Imports System.IO

Public Class ObjectDisposedExceptionTest
   
   Public Shared Sub Main()
      Dim ms As New MemoryStream(16)
      ms.Close()
      Try
         ms.ReadByte()
      Catch e As ObjectDisposedException
         Console.WriteLine("Caught: {0}", e.Message)
      End Try
   End Sub
End Class
'</Snippet1>

File: snippets/fsharp/System/ObjectDisposedException/Overview/objdispexc.fs
module objdispexc

//<Snippet1>
open System
open System.IO

let ms = new MemoryStream 16
ms.Close()
try
    ms.ReadByte()
    |> ignore
with :? ObjectDisposedException as e ->
   printfn $"Caught: {e.Message}"
//</Snippet1>